composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/webpack-encore-bundle",
+    "description": "Integration with your Symfony app & Webpack Encore!",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\WebpackEncoreBundle\\": "src"
```

Comment:
Shouldn't there be a trailing `/` after src?

Reply:
not needed. Composer would trim it if it exists.